.github/workflows/install-bindist.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Btw, it's a myth that this is posix and any better than `/bin/bash`, mostly incorrectly spread by nix users ;)

But any is fine.

Reply:
I am definitely guilty of cargo culting stuff in bash scripts, glad it is at least not incorrect :smile: